service/src/main/java/com/fecklessweasel/service/objectmodel/Class.java

```diff
@@ -0,0 +1,62 @@
+package com.fecklessweasel.service.objectmodel;
+
+import java.sql.Connection;
+
+import com.fecklessweasel.service.datatier.ClassTable;
+
+/**
+ * Stores all information about a class at a university.
+ * 
+ * @author Elliot Essman
+ */
+
+public class Class {
+
+	private long id;
+	private int deptId;
+	private int univId;
+	private int classNum;
+
+	private static int NUM_MAX = 999;
+	private static int NUM_MIN = 90;
+
+	private Class(long id, int univId, int deptId, int classNum) {
+		this.id = id;
+		this.univId = univId;
+		this.deptId = deptId;
+		this.classNum = classNum;
+	}
+
+	/**
+	 * Creates a class in the database
+	 * @param conn A connection to the database
+	 * @param univid Id of the university this class is at
+	 * @param deptid The official name of the department
+	 * @param classnum The number of this class
+	 * @return A class object
+	 */
+	public static Class create(Connection conn, int univId, int deptId, int classNum) throws ServiceException{
```

Comment:
Just one thought, while designing our object model ideally we should be able to perform transactions by identifying Classes, Departments, etc. by their objects rather than just IDs. In many cases there is no reason to expose the ID past the object model at all (think users, users have uid INT, but they are ID-ed above that by a string).

Might be worth modifying the design slightly but it's up to you and the team.